buildlog.txt

```diff
@@ -0,0 +1,2820 @@
+c:\Vagrant\boxes\shared\lwotc\
```

Comment:
As I think you're aware, this shouldn't be in the commit. Same with the Alien Base plots (no idea why they're different).

Reply:
Yeah my bad on the .txt file. Forgot to remove that. As with the base plot, I think that those files are not part of the .gitignore file in the content directory. I can change that if needed.

Reply:
The plots are explicitly excluded from gitignore because they are stored in the repository. I just don't know why you have "modified" versions. I assume by accident.

Reply:
Removed this file from repo.